setup.py

```diff
@@ -49,9 +49,11 @@
     ],
     'postgresql': [
         'invenio-db[postgresql,versioning]>=1.0.0a9',
+        'psycopg2',
```

Comment:
yep, as long as you have it too in the install_requires, it will pick up the first appearance. You can easily try it out adding a line to the install_requires with 'inspire-db[mysql]>=1.0.0a9' and moving it before and after the 'inspire-db[versioning]' one.